core/dbt/cli/main.py

```diff
@@ -213,6 +213,7 @@ def build(ctx, **kwargs):
 @p.profiles_dir
 @p.project_dir
 @p.target
+@p.target_path
```

Comment:
I don't believe the clean task uses `target_path`. It writes neither a results artifact nor a manifest.

Reply:
I was basing this off of the following in `project.py`:

```python
        flag_target_path = str(flags.TARGET_PATH) if flags.TARGET_PATH else None
        target_path: str = flag_or(flag_target_path, cfg.target_path, "target")
        log_path: str = str(flags.LOG_PATH)

        clean_targets: List[str] = value_or(cfg.clean_targets, [target_path])
```

Plus the fact that `CleanTask` refers to `self.project.clean_targets` and `self.config.clean_targets`.

Additionally, the command outputs `Checking target/*` and `Cleaned target/*` when I run it.

Let me know if I'm misunderstanding something though, but I would have guessed based on all this that it is using the target path.